src/plugins/gmx/gmxplugin.cpp

```diff
@@ -110,16 +110,87 @@ bool GmxPlugin::write(const Map *map, const QString &fileName)
     writeProperty(stream, map, "speed", 30);
     writeProperty(stream, map, "persistent", false);
     writeProperty(stream, map, "clearDisplayBuffer", true);
+    writeProperty(stream, map, "clearViewBackground", false);
+    // Check if views are defined
+    bool enableViews = false;
+    LayerIterator iterator(map);
+    while (const Layer *layer = iterator.next()) {
+
+        if (layer->layerType() != Layer::ObjectGroupType)
+            continue;
+
+        const ObjectGroup *objectLayer = static_cast<const ObjectGroup*>(layer);
+
+          for (const MapObject *object : objectLayer->objects()) {
+            const QString type = effectiveObjectType(object);
+            if (type == "view") {
+                enableViews = true;
+                break;
```

Comment:
This break only aborts the inner loop, but it will continue iterating the other layers. Can be solved by either another check whether `enableViews` is already true and then a `break`, or by extracting this "check if there are any views" out into a function (my preference).